src/components/cc-product-card/cc-product-card.js

```diff
@@ -0,0 +1,187 @@
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { i18n } from '../../lib/i18n.js';
+import '../cc-img/cc-img.js';
+import '../cc-button/cc-button.js';
+import '../cc-badge/cc-badge.js';
+
+/**
+ * @typedef {import('./cc-product-card.types.js').Keyword} Keyword
+ */
+
+/**
+ * A component displaying some information about a product in a card.
+ *
+ * @cssdisplay block
+ *
+ */
+export class CcProductCard extends LitElement {
+
+  static get properties () {
+    return {
+      description: { type: String },
+      iconUrl: { type: String, attribute: 'icon-url' },
+      keywords: { type: Array },
+      name: { type: String },
+      url: { type: String },
+    };
+  }
+
+  constructor () {
+    super();
+
+    /** @type {string|null} The description of the product */
+    this.description = null;
+
+    /** @type {string|null} The icon url of the product */
+    this.iconUrl = null;
+
+    /** @type {Keyword[]} The keywords representing the product */
+    this.keywords = [];
+
+    /** @type {string|null} The name of the product */
+    this.name = null;
+
+    /** @type {string|null} The url where the user will be redirected when the product is selected  */
+    this.url = null;
+  }
+
+  _areKeywordsAllHidden () {
+    return this.keywords.every((k) => k.hidden);
+  }
+
+  render () {
+
+    const hasKeywords = this.keywords.length > 0 && !this._areKeywordsAllHidden();
+
+    // language=HTML
+    return html`
+      <div class="wrapper ${classMap({ 'wrapper--no-keywords': !hasKeywords })}">
+        <cc-img src="${this.iconUrl}"></cc-img>
+        <span class="name">${this.name}</span>
+        <p class="description">${this.description}</p>
+        ${hasKeywords ? html`
+          <ul class="keywords">
+            ${this.keywords.map((keyword) => {
+              return (!keyword.hidden) ? html`
+                <li>
+                  <cc-badge class="keyword-badge">${keyword.value}</cc-badge>
+                </li>
+              ` : '';
+            })}
+          </ul>
+        ` : ''}
+        <a href="${this.url}" class="select">${i18n('cc-product-card.select')}</a>
+      </div>
```

Comment:
**fix:** the hidden link works for screen reader users but it causes issues for people browsing with voice only.

The link is called "select" but there is not visible text to guess that you have to say "select" to activate this card.

The best option would be to remove this hidden link and use a link on the heading instead.
We can still make it span through the whole card with a similar technique, we can discuss it in sync if you want.

If we do this, people may activate this link just by saying their command ("activate", "click", whatever) and the name of the product "NodeJS" for instance.
Sorry that I forgot about this subject when we discussed it in sync the other day :sweat_smile: 
```suggestion
      <div class="wrapper ${classMap({ 'wrapper--no-keywords': !hasKeywords })}">
        <cc-img src="${this.iconUrl}"></cc-img>
        <a href="${this.url}" class="name">${this.name}</a>
        <p class="description">${this.description}</p>
        ${hasKeywords ? html`
          <ul class="keywords">
            ${this.keywords.map((keyword) => {
              return (!keyword.hidden) ? html`
                <li>
                  <cc-badge class="keyword-badge">${keyword.value}</cc-badge>
                </li>
              ` : '';
            })}
          </ul>
        ` : ''}
      </div>
```